spec/lcm/integration/m_to_n_spec.rb

```diff
@@ -66,6 +66,13 @@
 
   after(:all) do
     @project.delete
+     begin
```

Comment:
Similar code found in m_to_n_spec.rb:69, visualisation_object_transfer_spec.rb:120, visualisation_object_transfer_spec.rb:127 (mass = 93)